ensime_shared/launcher.py

```diff
@@ -163,6 +163,13 @@ def _start_process(self, classpath):
         null = open(os.devnull, "r")
         java = os.path.join(self.config['java-home'], 'bin', 'java')
 
+        if(os.name == "nt"):
+            java += ".exe" #the java executable has the .exe extension in windows
+            # windows uses ';' instead of ':' for the classpath sep since ':' is reserved (C:\)
+            # for example  "c:\ensime.jar:c:\scala.jar"  =>  "c:\ensime.jar;c:\scala.jar"
+            # the following ".jar:" hack works because only jar's happen to be on the ensime classpath
+            classpath = classpath.replace(".jar:",".jar;")
```

Comment:
I think the right place to fix that is in the sbt plugin, in vim we should use the setting as it is. What do you think @fommil ?

I'm really surprised that this is the first time that supporting windows comes up.